playground/terraform/README.md

```diff
@@ -28,14 +28,31 @@ The following items need to be setup for the Playground cluster deployment on GC
 
 # Deployment steps
 
+## 0. Create GCS bucket for state
+
+```bash
+$ gsutil mb -p ${PROJECT_ID} gs://state-bucket-name
+$ gsutil versioning set on gs://state-bucket-name
+```
+
 ## 1. Provision infrastructure
 
 To deploy Playground infrastructure follow [README.md](./infrastructure/README.md) for infrastructure module.
 
-## 2. Build containers
+```bash
+./gradlew playground:terraform:InitInfrastructure -Pproject_environment="env-name"
+```
+
+## 2. Deploy application
 
-TBD
+```bash
+gcloud auth configure-docker us-central1-docker.pkg.dev
+  ```
 
-## 3. Deploy application
+```bash
+./gradlew playground:terraform:deployBackend  -Pproject_environment="env-name"
```

Comment:
I was struggling finding what the correct value to provide instead of `env-name`.  Would it help to link to another configuration file to provide context?

Reply:
I've updated readme